### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
Added documentation with latest format

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,87 @@
...
+
+### Introduction
+
+*Data Leakage Threat Response Solution Pack* is designed to provide a set of investigation and utility playbooks to respond to data leakage.
+
+Configure Cloud Access Security Broker(CASB) connector such as Symantec CloudSOC to capture events of policy violation like 'External File Sharing'. It creates an alert of type 'Data Leakage', and then triggers a response workflow.
+
+Refer to Simulation Scenarios - Data Leakage to experience the use case without CASB configuration
+
+This article describes the “Data Leakage Threat Response” Solutions Pack. The solution pack demonstrates the scenarios and use cases around Data Leakage.
```
Comment (cs-suyog-jinturkar) [inline README.md]: This article describes the “Data Leakage Threat Response” Solutions Pack. The solution pack demonstrates the scenarios and use cases around Data Leakage -> This is a repeated text. it is already mentioned above.

File: README.md [hunk 2]
```diff
@@ -1 +1,87 @@
...
+The below scenario demonstrates and generates a demo alert for the Alert.
```
Comment (cs-suyog-jinturkar) [inline README.md]: The below scenario demonstrates and generates a demo alert for the Alert. --> for the Alert?

Comment (cs-suyog-jinturkar) [reply README.md]: Please correct the sentence

File: README.md [hunk 3]
```diff
@@ -1 +1,87 @@
...
+
+- An alert is generated once detected a user shares a file externally, which is a violation of policy
```
Comment (cs-suyog-jinturkar) [inline README.md]: Replace this line with "An alert is generated when a policy gets violated due to file shared externally by the user"

File: README.md [hunk 4]
```diff
@@ -1 +1,87 @@
...
+- This triggers an investigation that determines if the employee is on the watchlist and if the file being shared is a sensitive document
```
Comment (cs-suyog-jinturkar) [inline README.md]: Who triggers the investigations? manual or automatic?

File: README.md [hunk 5]
```diff
@@ -1 +1,87 @@
...
+- In case a file is found to be a sensitive document then deactivate the user in AD and revoke file permission
```
Comment (cs-suyog-jinturkar) [inline README.md]: AD --> Microsoft Active Directory

File: README.md [hunk 6]
```diff
@@ -1 +1,87 @@
...
+- Search the incident in DLP and fetch the incident details. Create additional alerts if needed
```
Comment (cs-suyog-jinturkar) [inline README.md]: Search in DLP? DLP is a Data leakage Policy. How can user search that in a DLP?

Comment (cs-suyog-jinturkar) [reply README.md]: Who creates the additional alerts? and who will search them?

File: README.md [hunk 7]
```diff
@@ -1 +1,87 @@
...
+|SOAR Framework 1.0.0|Require for Incident Response modules and Action playbooks|[Click here](https://github.com/fortinet-fortisoar/solution-pack-soar-framework/blob/develop/README.md)|
+|SOC Simulator 1.0.1|Require for Scenario Module and SOC Simulator connector| [Click here](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/README.md)|
+
+## Contents
+
+1. Connector(s)
+
+    |**Sr.No.**|**Connector**|
+    | :- | :- |
+    |1|Active Directory|
```
Comment (cs-suyog-jinturkar) [inline README.md]: Microsoft Active ...

File: README.md [hunk 8]
```diff
@@ -1 +1,87 @@
...
+    |3|Symantec DLP|
+
+    **Warning:** After deployment, this Solution Pack will install or upgrade the stated connectors list.
+
+2. Modules Schema(s)
+    - Employee Watchlist
+    - Sensitive Files
+
+3. Role(s)
+    - Full App Permissions
```
Comment (cs-suyog-jinturkar) [inline README.md:66]: Why this SP needs a full app permission?

File: docs/solution-pack-guide.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+# Usage of Data Leakage Threat Response Solution Pack
+
+## Use Case Workflow
```
Comment (cs-suyog-jinturkar) [inline docs/solution-pack-guide.md]: I believe we don't need this file and its content. Please check with Sujit

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comments

File: docs/solution-pack-guide.md (hunk 9 above)
Comment (cs-rohit-khune) [reply docs/solution-pack-guide.md]: Discussed with Sujit, removed the files

File: README.md (hunk 1 above)
Comment (cs-rohit-khune) [reply README.md]: fixed it. Please check

File: README.md (hunk 2 above)
Comment (cs-rohit-khune) [reply README.md]: IMO we do not need this info as the same information is also given under the Usage section. Removing it altogether. Please review

File: README.md (hunk 3 above)
Comment (cs-rohit-khune) [reply README.md]: IMO we do not need this info as the same information is also given under the Usage section. Removing it altogether. Please review

File: README.md (hunk 4 above)
Comment (cs-rohit-khune) [reply README.md]: IMO we do not need this info as the same information is also given under the Usage section. Removing it altogether. Please review

File: README.md (hunk 5 above)
Comment (cs-rohit-khune) [reply README.md]: IMO we do not need this info as the same information is also given under the Usage section. Removing it altogether. Please review

File: README.md (hunk 6 above)
Comment (cs-rohit-khune) [reply README.md]: IMO we do not need this info as the same information is also given under the Usage section. Removing it altogether. Please review

File: README.md (hunk 7 above)
Comment (cs-rohit-khune) [reply README.md]: Done

File: README.md (hunk 8 above)
Comment (cs-rohit-khune) [reply README.md:66]: In this solution pack, the below modules are added. Required Full app permission role for the same

    - Employee Watchlist
    - Sensitive Files

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**